shared/search/user-search/render.desktop.js

```diff
@@ -120,9 +120,9 @@ export function Result ({result, searchText, onClickResult}: {result: SearchResu
 class Render extends Component<void, Props, void> {
   render () {
     return (
-      <Box style={{overflowY: 'auto', flex: 1}}>
+      <Box style={{overflowY: 'auto', flex: 1, position: 'relative', minHeight: 40}}>
         {this.props.waiting && <Icon type='icon-progress-grey-animated'
-          style={{position: 'absolute', top: 20, left: 0, right: 0, marginLeft: 'auto', marginRight: 'auto'}} />}
+          style={{position: 'absolute', width: 20, top: 0, left: 0, right: 0, marginLeft: 'auto', marginRight: 'auto'}} />}
```

Comment:
Is the `ProgressIndicator` component usable here? The animated icon will cause inconsistency in visdiff; `ProgressIndicator` includes logic to switch to a static version for screenshots.